Scripts/summarizeXSEC.py

```diff
@@ -3,49 +3,83 @@
 import numpy
 import ROOT
 import math
+import argparse
+
+
+
+parser=argparse.ArgumentParser()
+#parser.add_argument("-h", "--help", help="Display this message.")
+parser.add_argument("-f", "--filename", default="",  help="Filename")
+parser.add_argument("-u", "--units", default="Barn", help="Barn, mB, uB (Default:  Barn)")
+parser.add_argument("-l", "--label", default="PCC",  help="Label for file (Default:  PCC)")
+parser.add_argument("-r", "--range", default="",  help="Range for y-axis of plots (Default:  none)")
+parser.add_argument("-a", "--allbcids", default=0,  help="Use all bcids (Default:  0)")
+parser.add_argument("--plotlabel",  default="2016  (13TeV)", help="Label for the upper right.  Default:  2016  (13TeV)")
+args=parser.parse_args()
+
+
 
 ROOT.gROOT.SetBatch(ROOT.kTRUE)
 
-filename=sys.argv[1]
-file=open(filename)
+file=open(args.filename)
 fits=pickle.load(file)
 
 xsecInd=fits[0].index('xsec')
 xsecErrInd=fits[0].index('xsecErr')
 
-PCCBCIDs=["644",'1215','2269','2389','2589']
+BCIDs=["644",'1215','2269','2389','2589']
```

Comment:
Add an if statement, e.g. 

if '5TeV' in args.plotlabel:
 BCIDs = ["644",'1215','2269','2389','2589']

Or we can define a new argument to be parsed for the energy itself